src/session.cpp

```diff
@@ -326,6 +329,7 @@ std::size_t olm::Session::encrypt(
             message
         );
         olm::store_array(writer.one_time_key, bob_one_time_key.public_key);
+        olm::store_array(writer.prekey, bob_prekey.public_key);
```

Comment:
Wondering how things were working before this change... isn't it necessary to communicate the prekey in the message? In other words, wouldn't line 153 fail without this line?

Reply:
It is necessary to communicate the prekey (or its ID) in the first message. Line 153 should fail to load an array without this change, and I suspect Olm was failing silently.